content/logs/processing.md

```diff
@@ -159,6 +159,30 @@ These settings:
 Give the following results:
 {{< img src="logs/processing/useragent_processor.png" alt="Useragent processor" responsive="true" popup="true">}}
 
+## Technical limits
+
+To make sure the Log Management solution functions in an optimal way we set the following technical limits and rules to your log events as well as to some product features. These have been designed so you never reach them. However, if this is the case please read the actions we enforce and please contact us if you have any suggestion.
+
+### Limits applied to ingested log events
+
+* The size of a log event should not exceed 25K characters.
+* Log events can be submitted up to 6h in the past and 2h in the future into an index.
+* A log event once converted to JSON format should contain less than 256 attributes, each of those attribute’s key should be less than 50 characters, be nested in less than 10 successive levels and their respective value be less than 1024 characters if promoted as a facet.
+* A log event should not have more than 100 tags and each tag key should not exceed 50 characters and their respective value should not exceed 100 characters for a maximum of 10 millions unique tag per day.
```

Comment:
could we say that tags should not exceed `256 chars` instead? no need to separate key and value